### PR TITLE
Adding rake task to migrate state property to Sipity

### LINKED ISSUE #955: Rake task to migrate state to Sipity

### PR DESCRIPTION
**Pull request by escowles**
Fixes #955

### CODE REVIEW

**Comment by eliotjordan** [comment]
The task should probably set the resource.state to `nil` or `Vocab::FedoraResourceStatus.active`. Otherwise, CC is going to hide them default.

**Comment by tpendragon** [comment]
We should pull all the IDs from Solr and then `ActiveFedora::Base.find` each one, so we don't have to hold all those objects in memory.

File: lib/tasks/state.rake [hunk 1]
```diff
@@ -0,0 +1,16 @@
+namespace :state do
+  desc "Migrate state from the state property to Sipity"
+  task migrate: :environment do
+    ['MultiVolumeWork', 'ScannedResource', 'ImageWork', 'RasterWork', 'VectorWork'].each do |model|
+      ActiveFedora::SolrService.get("has_model_ssim:#{model}")['response']['docs'].each do |doc|
```
Comment (tpendragon) [inline lib/tasks/state.rake]: Simpler/faster would be
```
ActiveFedora::SolrService.query("has_model_ssim:#{model}", fl: "id").map{|x| x["id"]}.each do |id|
```

Comment (escowles) [inline lib/tasks/state.rake]: 👍  — I also noticed that I had `book_works` hardcoded, so I replaced that with a hash of workflows to models they contain.

**Pull request merged**